### PR TITLE
Dev ayush

### PR DESCRIPTION
**Pull request by ayushjidhanta**
I have heartly done my work please approve my pr.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-redux": "^8.0.5",
-    "react-router-dom": "^6.8.1",
+    "react-router-dom": "^6.22.1",
```
Comment (bhupenderhere) [inline package.json]: kindly change the version to older one

File: src/assets/PageNotFound/PageNotFound.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.not-found-container {
```
Comment (bhupenderhere) [inline src/assets/PageNotFound/PageNotFound.css]: kindly use module.css instead of plain css

File: src/components/Auth/SignIn/SignIn.jsx [hunk 3]
```diff
@@ -58,6 +59,7 @@ const SignIn = () => {
 
     try {
       const response = await signIn(credentials);
+      await new Promise((resolve) => setTimeout(resolve, 4000));
```
Comment (bhupenderhere) [inline src/components/Auth/SignIn/SignIn.jsx]: remove the timeout

File: src/components/Auth/SignUp/SignUp.jsx [hunk 4]
```diff
@@ -54,11 +56,19 @@ const SignUp = () => {
       );
       return;
     }
-
+     // disabling the action on signup btn 
+     if (e._reactName === "onClick") {
+      setSpinner(true);
+      setTimeout(() => {
+      showAlert("We are Sorry! Only Admins can make new accounts Currently we are declining new signup's");
+      setSpinner(false);
```
Comment (bhupenderhere) [inline src/components/Auth/SignUp/SignUp.jsx]: remove this check as user limit is handled

File: src/components/Auth/SignUp/SignUp.jsx [hunk 5]
```diff
@@ -54,11 +56,19 @@ const SignUp = () => {
...
+      showAlert("We are Sorry! Only Admins can make new accounts Currently we are declining new signup's");
+      setSpinner(false); 
+    }, 4000);
+      return ;
+    }
     setSpinner(true);
-
     try {
       const response = await signUp(credentials);
+      await new Promise((resolve) => setTimeout(resolve, 4000));
```
Comment (bhupenderhere) [inline src/components/Auth/SignUp/SignUp.jsx]: remove the timeout

File: package.json (hunk 1 above)
Comment (ayushjidhanta) [reply package.json]: The version is reset.

File: src/assets/PageNotFound/PageNotFound.css (hunk 2 above)
Comment (ayushjidhanta) [reply src/assets/PageNotFound/PageNotFound.css]: I have used module.css instead normal css file

**Review by bhupenderhere — APPROVED** [review]

**Pull request merged**